doc/classes/ConvexPolygonShape.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ConvexPolygonShape" inherits="Shape" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Convex Polygon Shape.
+		Convex Polygon Shape for 3D physics.
 	</brief_description>
 	<description>
 		Convex polygon shape resource, which can be set into a [PhysicsBody] or area.
```

Comment:
I don't like "set into" - I would change this to "assigned to" or "added to a [CollisionObject]."